src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -158,21 +161,27 @@ private void calculateDeadlines(Stage stage, CaseData caseData) {
         // Try and overwrite the deadline with inputted values from the data map.
         var overrideDeadline = caseData.getData(String.format("%s_DEADLINE", stage.getStageType()));
 
+        final StageTypeDto stageDefinition = infoClient.getAllStagesForCaseType(caseData.getType()).stream()
```

Comment:
This is duplicated from CaseDataService, maybe could extract to the infoClient class and call a single method there?

Reply:
After the changed I made to the other usage, they aren't duplicated anymore - this now the instance where the call is made and the stage type found in the same line.